h/templates/client/annotation.html

```diff
@@ -97,6 +97,13 @@
 </div>
 <!-- / Tags -->
 
+<div class="annotation-section small" ng-if="vm.editing">
+  <p ng-show="level.text == 'Only Me'">
```

Comment:
vm.isPrivate( ) works, but doesn't change when you change the privacy control, only on save.

![gif of this behavior](https://cloud.githubusercontent.com/assets/521978/8172311/45a9e670-1374-11e5-8b6c-b9c725d58797.gif)